src/sdk/pynni/nni/gridsearch_tuner/gridsearch_tuner.py

```diff
@@ -135,9 +139,31 @@ def update_search_space(self, search_space):
 
     def generate_parameters(self, parameter_id):
         self.count += 1
-        if self.count > len(self.expanded_search_space)-1:
-            raise nni.NoMoreTrialError('no more parameters now.')
-        return self.expanded_search_space[self.count]
+        while (self.count <= len(self.expanded_search_space)-1):
```

Comment:
@Crysple please help review the modification of grid search

Reply:
Done